packages/cli/test/functional/test_site/testMermaid.md

```diff
@@ -0,0 +1,71 @@
+**Mermaid Test**
```

Comment:
Could we add a functional test to make sure that mermaid works when using include?

Reply:
Currently, the mermaid plugin loads after the two internal and external plugin we have on the test site, so it will like this at first..
![image](https://github.com/MarkBind/markbind/assets/77204346/1c49e5d8-1128-4888-8b33-f8834f334b35)

Reply:
> Currently, the mermaid plugin loads after the two internal and external plugin we have on the test site, so it will like this at first.. ![image](https://private-user-images.githubusercontent.com/77204346/322904987-1c49e5d8-1128-4888-8b33-f8834f334b35.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTMyODE3MTQsIm5iZiI6MTcxMzI4MTQxNCwicGF0aCI6Ii83NzIwNDM0Ni8zMjI5MDQ5ODctMWM0OWU1ZDgtMTEyOC00ODg4LThiMzMtZjg4MzRmMzM0YjM1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDE2VDE1MzAxNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWY0Zjc2MmVkNjY3NmJmYTVhN2I5YWNiNzMxYTNiMzgwMDIzMWZlOWU1ZmY0NTE5ODQwYzk0OTBhYjNjYzI3MTkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.JBNz1OaW2Cvt8pymJvEK6azNQYkyMmAMOXT0J95Kjoc)

Thanks for the investigation!

Reply:
HI @Tim-Siu can you add an exmaple of mermaid using includes?

Reply:
Sorry, I mistook it for a unit test and was thinking about how to achieve this. Thank you for your reminder!